doc/sphinxman/source/sapt.rst

```diff
@@ -532,9 +532,13 @@ estimated by scaling from the uncoupled value either by a fitted fixed
 value as suggested by [Hesselmann:2014:094107]_, or by the ratio of
 coupled and uncoupled dispersion energy. This can be controlled by
 keyword |sapt__sapt_dft_exch_disp_scale_scheme|, with ``FIXED`` being
-the former approach, ``DISP`` being the latter and ``NONE`` for not
+the former approach (default option), ``DISP`` being the latter and ``NONE`` for not
 scaling and use the uncoupled exchange-dispersion energy directly.
 
+.. warning:: The default option of |sapt__sapt_dft_exch_disp_scale_scheme|
+             has been changed from ``DISP`` to ``FIXED``. This might cause
+             an older version of |PSIfour| to obatin a different value of
+             exchange-dispersion energy from the latest version.
```

Comment:
There also needs to be a warning about the change to "SAPT_DFT_EXCH_DISP_FIXED_SCALE", both here in the documentation and in the output file.